opds.py

```diff
@@ -1494,16 +1499,6 @@ def add_breadcrumbs(self, lane, include_lane=False, entrypoint=None):
                 )
             )
 
-        # Include link to the lane itself.
-        # For search, breadcrumbs include the searched lane
-        if include_lane:
-            breadcrumbs.append(
-                AtomFeed.link(
-                    title=lane.display_name,
-                    href=annotator.lane_url(lane) + entrypoint_query
-                )
-            )
-
```

Comment:
We don't need this code anymore since `include_lane` now makes sure the lane itself goes into `usable_parentage`. It's then handled like all the other lanes as we iterate over `usable_parentage`.